lte/gateway/release/build-magma.sh

```diff
@@ -126,11 +126,17 @@ OAI_DEPS=(
     "liblfds710"
     "magma-sctpd >= ${SCTPD_MIN_VERSION}"
     "libczmq-dev >= 4.0.2-7"
+    "oai-gtp >= 4.9.5"
     )
 
 # OVS runtime dependencies
 OVS_DEPS=(
     "magma-libfluid >= 0.1.0.4"
+    "openvswitch-common >= 2.8.9"
+    "libopenvswitch >= 2.8.9"
+    "openvswitch-switch >= 2.8.9"
+    "openvswitch-datapath-source-4.9.0-9 >= 2.8.9"
+    "openvswitch-datapath-dkms >= 2.8.9"
```

Comment:
I thought there was a specific reason why we chose to remove OVS as a magma dependency, although I can't remember anymore - @amarpad?

Reply:
I think the reason was that due to the kernel API changing within the same release it pulled in a version of ovs from our repo that didn't work for some of the installs due to the mismatched kernel.
For some of the other installs with older ovses forcing the upgrade of the kernel also didn't work as there the kernel was too old for the new ovs kernel module
All in all it broke on newer and older kernels.